src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -66,6 +71,8 @@ public class InternalUsersApiAction extends PatchableResourceApiAction {
             // corrected mapping, introduced in OpenSearch Security
             new Route(Method.GET, "/internalusers/{name}"),
             new Route(Method.GET, "/internalusers/"),
+            new Route(Method.GET, SERVICE_ACCOUNTS_ENDPOINT),
+            new Route(Method.GET, INTERNAL_ACCOUNTS_ENDPOINT),
```

Comment:
I didn't want to remove/change any existing endpoints   to avoid backward compatibility issues and because the issue description mentioned "adding endpoints for those". 

_As a follow-up, we can a modify /internalusers handler to check for a query parameter called filter_by and it can have either of two values service-accounts & internal-accounts. This way we avoid creating new endpoints._


If editing the existing endpoint is ok, i think this makes sense

Reply:
updated to use the existing endpoint. Will update the Dashboard plugins accordingly